### ISSUE TITLE
Manually edit the path for jobs

### ISSUE THREAD

**Issue by llrs** [rstudio/rstudio#6036]
I am not sure if this is a bug or a feature request, but let me describe it:

I recently started using the Jobs feature of Rstudio. I can select the path of the Working Directory and of the R Script. At the moment the Working Directory selects the R folder so I need to jump to other folders and then make sure that it starts on the desired folder and not again on the R folder. If I just go to the upper folder it still selects the R folder:
![Screenshot from 2020-01-16 15-14-43](https://user-images.githubusercontent.com/6818218/72534549-4d750600-3877-11ea-9c49-3789d3da52aa.png)

I open the directory I am, but note that the upper navigation bar still is selecting the /R folder.

![Screenshot from 2020-01-16 15-46-49](https://user-images.githubusercontent.com/6818218/72534632-6ed5f200-3877-11ea-994b-32a3166e99cc.png)

 
However, I need to do it manually via Browsing the folders. It would be nice if I could edit it manually instead of opening the file browser. 

I am using Rstudio Desktop 1.2.5001 Build 93 (7b3fe265, 2019-09-18) on a Ubuntu 18.04.

**Comment by ronblum** [comment]
@llrs Thank you for raising this! I'll mark it as a bug to review for now, although as you point out, this might be an enhancement.

@dfalty Can you provide any insight to this?

**Comment by dfalty** [comment]
If I understand correctly if the path was just manually editable without having select `Browse` you would have what you need? If so, it's just an enhancement so that we allow that action as it's not editable now.

**Comment by gtritchie** [comment]
See also #5811

**Comment by llrs** [comment]
Yes, having it editable would solve my "problem", alternatively that it wouldn't select the `project/R` folder as default would work too.

**Comment by llrs** [comment]
Just stumbled with this again. Another (easy) option would be if the default is the *.Rproj directory not where the current R file is in.

**Comment by llrs** [comment]
This isn't fixed/addressed on Rstudio Version 1.4.1103. Would be nice to get it solved, or closed with an explanation. Thanks!

**Comment by jmiahjones** [comment]
Since you appear to be using Linux with the Nautilus browser (Ubuntu perhaps?), I can give some insight into the user experience here. Nautilus has the fun behavior that when you navigate one directory up, it selects the folder you just left. You can see this in your first screen-grab, where the R folder is highlighted. So when you click the Open button, you are still technically selecting the R directory. The workaround I have been using is to navigate two directories up so that the project folder is highlighted when you click Open, thus setting the project directory as the working directory. Other programs which use the default file browser on my Ubuntu box have this behavior as well.

Maybe this gives you a workaround until some better behavior is built into the Job creation dialog...

**Comment by llrs** [comment]
Hi @jmiahjones, yes I'm using Ubuntu. I've been using this workaround for a while. Thanks!

**Comment by llrs** [comment]
This is still happening (I know it is not a high priority, but if I can prevent to slip behind the radar I will), although the linked issue #8985 seems to have gathered more discussion on this.
